app/code/Magento/LoginAsCustomerGraphQl/Model/Resolver/RequestCustomerToken.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\LoginAsCustomerGraphQl\Model\Resolver;
+
+use Magento\Framework\AuthorizationInterface;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Query\Resolver\Value;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Query\Resolver\ContextInterface;
+use Magento\Framework\GraphQl\Exception\GraphQlNoSuchEntityException;
+use Magento\Framework\GraphQl\Exception\GraphQlAuthorizationException;
+use Magento\LoginAsCustomerApi\Api\ConfigInterface as LoginAsCustomerConfig;
+use Magento\LoginAsCustomerGraphQl\Model\LoginAsCustomer\CreateCustomerToken;
+
+/**
+ * Gets customer token
+ */
+class RequestCustomerToken implements ResolverInterface
+{
+    /**
+     * @var LoginAsCustomerConfig
+     */
+    private $config;
+
+    /**
+     * @var AuthorizationInterface
+     */
+    private $authorization;
+
+    /**
+     * @var CreateCustomerToken
+     */
+    private $createCustomerToken;
+
+    /**
+     * RequestCustomerToken constructor.
+     * @param AuthorizationInterface $authorization
+     * @param LoginAsCustomerConfig $config
+     * @param CreateCustomerToken $createCustomerToken
+     */
+    public function __construct(
+        AuthorizationInterface $authorization,
+        LoginAsCustomerConfig $config,
+        CreateCustomerToken $createCustomerToken
+    ) {
+        $this->authorization = $authorization;
+        $this->config = $config;
+        $this->createCustomerToken = $createCustomerToken;
+    }
+
+    /**
+     * Get Customer Token using email
+     *
+     * @param Field $field
+     * @param ContextInterface $context
+     * @param ResolveInfo $info
+     * @param array|null $value
+     * @param array|null $args
+     * @return Value|mixed|void
+     * @throws GraphQlAuthorizationException|GraphQlNoSuchEntityException|LocalizedException
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    public function resolve(
+        Field $field,
+        $context,
+        ResolveInfo $info,
+        array $value = null,
+        array $args = null
+    ) {
+        $isAllowedLogin = $this->authorization->isAllowed('Magento_LoginAsCustomer::login');
+        $isEnabled = $this->config->isEnabled();
+
+        /* Get input params */
+        try {
+            $args = $args['input'] ?: [];
```

Comment:
Please, correct me if I'm wrong, but there's no chance of getting `NoSuchEntityException` from this expression. Instead of try->catch we may use a simple condition and throw `GraphQlInputException` in case of missing argument.